source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/ChargeCommands/Handlers/ChargeIntegrationEventsPublisher.cs

```diff
@@ -39,7 +40,7 @@ public async Task PublishAsync(ChargeCommandAcceptedEvent chargeCommandAcceptedE
 
             foreach (var chargeOperationDto in chargeCommandAcceptedEvent.Command.ChargeOperations)
             {
-                await _chargePublisher.PublishChargeCreatedAsync(chargeOperationDto).ConfigureAwait(false);
+                await _chargeInformationPublisher.PublishChargeInformationCreatedAsync(chargeOperationDto).ConfigureAwait(false);
```

Comment:
Suggestion: keep the method name as it was, to reflect the [contract](https://github.com/Energinet-DataHub/geh-charges/blob/main/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/Contracts/Public/ChargeCreated/ChargeCreated.proto) it is publishing.
```suggestion
                await _chargeInformationPublisher.PublishChargeCreatedAsync(chargeOperationDto).ConfigureAwait(false);
```

Reply:
ok